src/gitHubIssueFetcher.ts

```diff
@@ -68,6 +70,14 @@ query findByLabel($queryString:String!, $pageSize:Int, $after:String) {
                         nameWithOwner
                     }
                     url
+                    updatedAt
+                    labels(first: $pageSize) {
```

Comment:
I'm not sure this should reuse the $pageSize value. I'm thinking it should either just be hard-coded at 100, or (probably better) use a new const like `maxLabelsToRetrieve`, which we don't need to make configurable, and could just be defined at the top of the file.

Reply:
Totally agree. Updated this to a new const